Library/Source/RuntimeImpl.cpp

```diff
@@ -9,187 +9,190 @@
 
 namespace babylon
 {
-    RuntimeImpl::RuntimeImpl(void* nativeWindowPtr, const std::string& rootUrl)
```

Comment:
This looks like the spacing is funky. Leftovers from the namespace thing?

Reply:
Might be, I'll do a clang format before merging anyway.